contracts/callpaths/WarmPath.sol

```diff
@@ -327,42 +321,43 @@ contract WarmPath is MarketSequencer, SettleLayer, ProtocolAccount {
         return code == UserCmd.BURN_AMBIENT_LIQ_LP ||
                code == UserCmd.BURN_AMBIENT_BASE_LP ||
                code == UserCmd.BURN_AMBIENT_QUOTE_LP ||
-               code == UserCmd.HARVEST_LP;
+               code == UserCmd.HARVEST_LP ||
```

Comment:
we can save one check on this, if we rename it to isMint, 6 MINT checks will be there 
see if you wanna incorporate that